config/v1/types_apiserver.go

```diff
@@ -208,6 +225,13 @@ const (
 	// aesgcm refers to a type where AES-GCM with random nonce and a 32-byte key
 	// is used to perform encryption at the datastore layer.
 	EncryptionTypeAESGCM EncryptionType = "aesgcm"
+
+	// kms refers to a type of encryption where the encryption keys are managed
+	// outside the control plane in a Key Management Service instance,
+	// encryption is still performed at the datastore layer.
+	//
+	// +openshift:enable:FeatureGate=KMSv2
+	EncryptionTypeKMS EncryptionType = "kms"
```

Comment:
Enum values must be PascalCase, in this example, that means
```suggestion
	EncryptionTypeKMS EncryptionType = "KMS"
```